travis/createExtractTarball.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Creates the npm tarball using the npm pack command then extracts the contents to a temporary directory
+
+npm pack
+# provides compatibility for both macOS and Linux
+npmtmpdir=`mktemp -d 2>/dev/null || mktemp -d -t 'npmtmpdir'`
+
+tar -xvf nasaworldwind-worldwind-*.tgz -C $npmtmpdir --strip-components 1
+
+export NPM_TMP_DIR=$npmtmpdir
```

Comment:
This is too difficult to follow. It appears in the Travis file as an ambiguous variable, but it’s not till scanning this that I connected th dots. Could we extract to a predefined directory name in the project itself?

Reply:
With the difficulty in exporting the environment variable for the temporary directory, I believe the easiest path forward is exactly that, defining a directory name in the project.

Reply:
That sounds reasonable. Provided the name is clear and there’s no collision, that sounds like a good plan.